<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1073</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1073</width>
    <height>768</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>4</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>4</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Adelie</string>
  </property>
  <property name="windowOpacity">
   <double>4.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

	background-color: rgba(64, 213, 191, 0);
}
QMainWindow::separator {
    width: 10px;
    height: 0px;
  
    padding:100px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1073</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1073</width>
     <height>768</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1073</width>
      <height>786</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1073</width>
      <height>786</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1073</width>
      <height>786</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget{

	background-color: rgba(191, 64, 64, 0);
}</string>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_4"/>
    <widget class="QWidget" name="page_5"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGNU">
    <property name="title">
     <string>GNU</string>
    </property>
    <addaction name="actionGNU_Core_Utilities"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <addaction name="actionIfconfig"/>
    <addaction name="actionIwconfig"/>
    <addaction name="separator"/>
    <addaction name="actionNetstat"/>
    <addaction name="actionPing_4"/>
   </widget>
   <addaction name="menuGNU"/>
   <addaction name="menuNetwork"/>
  </widget>
  <action name="actionPing">
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionCore_Utilities">
   <property name="text">
    <string>Core Utilities</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCore_Utils">
   <property name="text">
    <string>Core Utilities</string>
   </property>
  </action>
  <action name="actionPing_2">
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionPing_menu">
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionifconfig">
   <property name="text">
    <string>ifconfig</string>
   </property>
  </action>
  <action name="actioniwconfig">
   <property name="text">
    <string>iwconfig</string>
   </property>
  </action>
  <action name="actionnetstat">
   <property name="text">
    <string>netstat</string>
   </property>
  </action>
  <action name="actionHash_Text">
   <property name="text">
    <string>Hash Text</string>
   </property>
  </action>
  <action name="actionReplace_Text">
   <property name="text">
    <string>Replace Text</string>
   </property>
  </action>
  <action name="actionCompare_Text">
   <property name="text">
    <string>Compare Text</string>
   </property>
  </action>
  <action name="actionSystem_Logging">
   <property name="text">
    <string>System Logging</string>
   </property>
  </action>
  <action name="actionImage_Tools">
   <property name="text">
    <string>Image Tools</string>
   </property>
  </action>
  <action name="actionSchedule_Scripts">
   <property name="text">
    <string>Schedule Scripts</string>
   </property>
  </action>
  <action name="actionPing_3">
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionIfconfig">
   <property name="text">
    <string>Ifconfig</string>
   </property>
  </action>
  <action name="actionIwconfig">
   <property name="text">
    <string>Iwconfig</string>
   </property>
  </action>
  <action name="actionNetstat">
   <property name="text">
    <string>Netstat</string>
   </property>
  </action>
  <action name="actionPing_4">
   <property name="text">
    <string>Ping</string>
   </property>
  </action>
  <action name="actionGNU_Core_Utilities">
   <property name="text">
    <string>GNU Core Utilities</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
